modules/asg_node_group/variables.tf

```diff
@@ -184,3 +184,9 @@ variable "bottlerocket_admin_container_source" {
   default     = ""
   description = "URI of a custom admin container image"
 }
+
+variable "nodes_metadata_http_tokens" {
```

Comment:
It might be more robust to have this be a bool, then use a ternary to choose one of the two valid strings..

i.e.

```hcl
http_tokens  = var.nodes_metadata_http_tokens ? "required", "optional"
```

Reply:
If we did that then the name should be something more like `require_instance_metadata_tokens`

Reply:
sorted out